verification/curator-service/ui/src/components/Profile.tsx

```diff
@@ -83,6 +400,7 @@ export default function Profile(): JSX.Element {
                                     throw Error(`Unknown role ${role}`);
                             }
                         })}
+                        {user.googleID === "" && <ChangePasswordFormInProfile />}
```

Comment:
Here you should actually be checking if there is `googleId` property in `user` object at all. If not than show password change form. Please change this condition to: `!user.googleID && <.../>`